vtr_flow/scripts/download_titan.py

```diff
@@ -228,14 +232,15 @@ def extract_to_vtr_flow_dir(args, tar_gz_filename):
                     dst_file_path = os.path.join(titan_benchmarks_extract_dir, filename)
                 elif fnmatch.fnmatch(src_file_path, "*/titan_release*/benchmarks/other_benchmarks/*/*/*.blif"):
                     dst_file_path = os.path.join(titan_other_benchmarks_extract_dir, filename)
-                elif filename.endswith(".xml"):
-
-                    if args.upgrade_archs:
-                        #Apply the Architecture XML upgrade script
-                        print("Upgrading architecture file:")
-                        os.system("{} {}".format(arch_upgrade_script, src_file_path))
-                    
-                    dst_file_path = os.path.join(titan_arch_extract_dir, filename)
+                #ignore the replacement of arch files
+                #elif filename.endswith(".xml"):
+
+                #    if args.upgrade_archs:
+                #        #Apply the Architecture XML upgrade script
+                #        print("Upgrading architecture file:")
+                #        os.system("{} {}".format(arch_upgrade_script, src_file_path))
+                #    
+                #    dst_file_path = os.path.join(titan_arch_extract_dir, filename)
```

Comment:
Since this is no longer necessary, its probably better to simply delete this (we can always get it back from the revision history if needed).